### ISSUE TITLE
can't access UITableViewRowAction with accessibilityLabel in iOS 11 any more.

### ISSUE THREAD

**Issue by sabagogolidze** [calabash/calabash-ios#1340]
before xCode and iOS update I was able to access UITableViewRowAction with accessibilityLabel.

I have several UITableViewRowAction in table view cell so I was adding accessibilityLabels on them and could access them by calabash-ios console query. now I can't.
UITableViewRowAction* editAction;
[editAction setAccessibilityLabel:@"edit_action_label"];
query("* marked:'view_action_label'")
before update this query was returning view  for UITableViewRowAction. now it returns empty array. Is there any other way to set some identifier on UITableViewRowAction and then accessing it? I have already tried accessibilityIdentifier and setIsAccessibilityElement:YES but it does not work :(

**Comment by jmoody** [comment]
Try asking in Gitter. 

See if you can find your view using a DeviceAgent query:

```
device_agent.query({all: true})
```

In the console, try using the `labels`, `ids`, `text`, `marks` commands.

```
$ be calabash-ios console
Running irb...

#########################  Useful Methods  ##########################
     ids => List all the visible accessibility ids.
  labels => List all the visible accessibility labels.
    text => List all the visible texts.
   marks => List all the visible marks.
    tree => The app's visible view hierarchy.
   flash => flash(<query>); Disco effect for views matching <query>
 verbose => Turn debug logging on.
   quiet => Turn debug logging off.
    copy => Copy console commands to clipboard.
   clear => Clear the console.

Calabash says, "Your wish is my command."
calabash-ios 0.21.1>
```

And don't forget about the `flash` command.

If you find the view that _contains_ the view, something like this might work:

```
# Touch the 2nd sub view in a row marked 'Run 5K'
touch("UITableViewCell marked:'Run 5K' descendant index:1")

# Touch the first label in the 2nd sub view in a row marked 'Run 5K'
touch("UITableViewCell marked:'Run 5K' descendant index:1 descendant UILabel")
```
